source/agora/common/Config.d

```diff
@@ -109,9 +109,10 @@ public struct NodeConfig
     /// in place of the built-in commons budget address as defined by CoinNet
     public PublicKey commons_budget_address;
 
-    /// If set, a hexdump serialized representation of the genesis block to use
-    /// in place of the built-in genesis block as defined by CoinNet
-    public string genesis_block;
+    /// If set to true will run in testing mode and use different
+    /// genesis block (agora.consensus.data.genesis.Test)
+    /// and TODO: addresses should be different prefix (e.g. TN... for TestNet)
+    public bool testing;
```

Comment:
On the long run, this should be an enum, not a boolean.

Reply:
With the name of the network?

Reply:
Yes. I had the following in mind:
```
enum Network
{
    MainNet,
    TestNet,
    Testing,
}
```

But actually, we could get away with having `TestNet` starting with a `Testing` block and just moving all funds away from the WK keys.

Reply:
As we discussed can remain as testing to cover unit / integration testing as well as TestNet. Just have to be careful that nobody steals all the TestNet coins as addresses are known. Can be prevented if we transfer enough to unknown address at the start.